Mage.Sets/src/mage/cards/d/DranaAndLinvala.java

```diff
@@ -0,0 +1,121 @@
+package mage.cards.d;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.ActivatedAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.RestrictionEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class DranaAndLinvala extends CardImpl {
+    public DranaAndLinvala(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{W}{W}{B}");
+        this.addSuperType(SuperType.LEGENDARY);
+        this.addSubType(SubType.VAMPIRE);
+        this.addSubType(SubType.ANGEL);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(4);
+
+        //Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        //Vigilance
+        this.addAbility(VigilanceAbility.getInstance());
+
+        //Activated abilities of creatures your opponents control can't be activated.
+        this.addAbility(new SimpleStaticAbility(new DranaAndLinvalaCantActivateEffect()));
+
+        //Drana and Linvala has all activated abilities of all creatures your opponents control. You may spend mana as
+        //though it were mana of any color to activate those abilities.
+        this.addAbility(new SimpleStaticAbility(new DranaAndLinvalaAllActivatedAbilitiesEffect()));
```

Comment:
is the mana effect missing or did I just miss it?

Reply:
seems like it is missing!